tests/tox.sh

```diff
@@ -36,11 +36,15 @@ fi
 # copy the files to the root for the
 # types of images we're going to build
 mkdir -p {base,daemon,demo}
-# starting with kraken, the base image does not exist
 cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/base/* base || true
 cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/daemon/* daemon
 cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/demo/* demo || true
 
+# remove the base dir any release after jewel 
```

Comment:
The check I added only build the image if base exists and not empty. In our case, it'll be empty so we're good.